src/handlers/event/analyze_logs/issues.rs

```diff
@@ -334,7 +334,7 @@ fn offline_launch(log: &str) -> Issue {
 			.to_string(),
 	);
 
-	let found = log.contains("(missing)\n");
+	let found = log.contains("(missing)\n") || log.contains("(missing)\r\n");
```

Comment:
If this conversion is done by Discord (I assume that's what you meant) why only here? Maybe it would be best to convert the whole file before processing...

Reply:
discord files use a `\r\n` yeah; sure i'll change it, that would prob make more sense